src/NJsonSchema/Annotations/IMultiJsonSchemaExtensionDataAttribute.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+using System.Collections.Generic;
+
+namespace NJsonSchema.Annotations
+{
+    /// <summary>Interface to add multiple extension data property to a class or property, implementation needs to inherit from System.Attribute.</summary>
+    public interface IMultiJsonSchemaExtensionDataAttribute
```

Comment:
I'm not a subject matter expect here (Rico is), but would it make sore sense just to make a breaking change to change the support to have dictionary in original interface so there wouldn't be two ways to achieve the same thing (adding one item can be done via dictionary or via old interface).

Reply:
this definitely could be done, I took the more conservative approach because I didn't want to break existing functionality. I don't mind refactoring to do what you suggest if that's what is agreed upon

Reply:
would appreciate a review @RicoSuter when you have some time